docs/lava-blockchain/join-testnet-manual.md

```diff
@@ -103,7 +128,7 @@ Note that it does **not** include the "Cosmovisor" tool, hence once you install
     lavad_binary_path="$HOME/go/bin/"
     mkdir -p $lavad_binary_path
     # Download the genesis binary to the lava path
-    wget https://lava-binary-upgrades.s3.amazonaws.com/testnet/v0.3.0/lavad
+    wget https://lava-binary-upgrades.s3.amazonaws.com/testnet/v0.12.1-hf/lavad
```

Comment:
this is not correct.
you have to use this file once you reach the problematic block, described in the slack channel.
if you start from genesis you start with v0.3.0 until you reach that block. than you swap for v0.12.1-hf